extensions/extensions-configuration-store-vault/pom.xml

```diff
@@ -86,11 +138,26 @@
             <groupId>com.boozallen.aissemble</groupId>
             <artifactId>foundation-configuration-store</artifactId>
             <version>${version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.quarkiverse.cucumber</groupId>
+                    <artifactId>quarkus-cucumber</artifactId>
```

Comment:
Q/S: Two thought occur to me: 1) Why are test dependencies being pulled here such that they have to be excluded, 2) why are these not the same version as is being used here (or are they being removed for some other reason)?

Reply:
Yeah.. the foundation-configuration-store module needs these two packages for its tests (quarkus-cucumber test), and, for this module, we don't need to start up a webserver so we don't need the quarkus-cucumber test. Also, these two transitive dependencies are causing this module's cucumber testing not being detected because of version conflicts.